src/core/server/logging/layouts/json_layout.test.ts

```diff
@@ -122,36 +128,131 @@ test('`format()` correctly formats error record with meta-data', () => {
   expect(
     JSON.parse(
       layout.format({
-        context: 'error-with-meta',
         level: LogLevel.Debug,
+        context: 'error-with-meta',
         error: {
           message: 'Some error message',
-          name: 'Some error name',
+          name: 'Some error type',
           stack: 'Some error stack',
```

Comment:
Shouldn't this be `stack_trace`?

Reply:
this is the mock input - a standard javascript error.  it'll get converted in the format function to stack_trace

Reply:
Ah, this is the input - gotcha.